Animal.java

```diff
@@ -0,0 +1,52 @@
+ class Animal {
+     String name;
+     int eyes;
+     int eat;
+     int drink;
+     void aminal(){
+         System.out.println("I am animal");
+     }
+     void eat(){
+         System.out.println("Animal eats");
+     }
+
+     void drink(){
+         System.out.println("Animal drinks");
+     }
+     static class Pet {
+         void pet(){
+             System.out.println("I am pet");
+             System.out.println("Pet runs");
+             System.out.println("Pet runs« jumps");
+         }
+         String name;
+         int tai = 1;
+         int paw = 4;
+         String run;
+         String jump;
+
+         class Dog {
```

Comment:
public class Dog extends Pet {
    Dog(String name) {
        super();
        this.name = name;
        System.out.println("I am a dog and my name is: " + name);
    }

    void play() {
        System.out.println("Dog plays");
    }

    public static void main(String[] args) {
        Dog dog = new Dog("Buddy");
        dog.play();
    }
}